src/saveload/saveload.h

```diff
@@ -1197,7 +1197,7 @@ inline constexpr bool SlCheckVarSize(SaveLoadType cmd, VarType type, size_t leng
  * @param variable Name of the global variable.
  * @param type     Storage of the data in memory and in the savegame.
  */
-#define SLEG_ARR(name, variable, type) SLEG_CONDARR(name, variable, type, lengthof(variable), SL_MIN_VERSION, SL_MAX_VERSION)
```

Comment:
why can't std::size be part of the define, rather than another param?

(if not, docs ;) )

Reply:
Consistency so it matches all the other arrays, just this one happened to not be used so it wasn't noticed. The length is specified in those cases where the length varies depending on saveload version. (It doesn't here, but it could.)